q2_demux/plugin_setup.py

```diff
@@ -41,4 +41,12 @@
                  'http://www.earthmicrobiome.org/emp-standard-protocols/')
 )
 
+plugin.visualizers.register_function(
+    function=q2_demux.summary,
+    inputs={'data': SampleData[SequencesWithQuality]},
+    parameters={},
+    name='Summarize counts per sample.',
+    description=('Generate a tabular summary of counts per sample.')
```

Comment:
Maybe add "from sequence data that has already been demultiplexed." to the end of the sentence